systemd/bootc-multi-user.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Dummy bootc service that activates during system boot
+Documentation=man:bootc(8)
+
+[Service]
+Type=oneshot
+ExecStart=/bin/true
```

Comment:
Not to make you retest all of this again, but I suspect this could be actually be a `.target` unit and not a `.service` right? The role of `.target` units is precisely for  synthetic ordering.

If we *were* to make this a service (or even a target)...I would bikeshed that we shouldn't call it `-multi-user` since that has an implication we have something to do with user logins.

How about `bootc-status-updated-onboot.target`?

Reply:
(If you have tested this PR as is and it all works, I am ok to just ship this and do further cleanup as a followup, to be clear)

Reply:
Yeah I like this better.  I was trying to avoid making the "dependent" services (rhsm and potential future others) be WantedBy multiple things, but the hacky dummy service it resulted in is not great.